manuskript/exporter/manuskript/plainText.py

```diff
@@ -43,9 +43,9 @@ def output(self, settingsWidget):
         settings = settingsWidget.getSettings()
         try:
             return self.concatenate(mainWindow().mdlOutline.rootItem, settings)
-        except re.error:
+        except re.error as e:
             QMessageBox.warning(mainWindow().dialog, qApp.translate("Export", "Error"),
-                                qApp.translate("Export", "Error processing regular expression"))
+                                qApp.translate("Export", "Error processing regular expression : \n%s" % str(e)))
```

Comment:
Wonder if it should be this instead : 
`      qApp.translate("Export", "Error processing regular expression : \n{}").format(str(e)))`

Reply:
Yes, I agree that it would be better to include the actual regexp error string but not translate it.  Only the static text contained in the Manuskript application should be translated.

Reply:
Ha! That was obvious but hadn't realized the error string would get translated too. I never worked with translations before so didn't know that would be the behaviour (makes sense now that I think about it).
I'm busy today, so feel free to change it and merge if you want, otherwise, I'll apply the change tomorrow.

Reply:
I don't think the regexp error string will get translated in either situation.  However I do prefer the latter change thought because it more closely matches the existing code.  For example see https://github.com/olivierkes/manuskript/blob/9b774d327f8ef83c729f875667a074988b366f8d/manuskript/models/references.py#L495

Also an update tomorrow will be fine.  If I make the change and push it then it breaks some of the automatic tracking and accreditation that github performs.  Hence I prefer for the original author to make the change, get the credit, and have github mark the PR as merged.